Sources/ParseLiveQuery/Client.swift

```diff
@@ -149,13 +150,18 @@ extension Client {
             )
             subscriptions.append(subscriptionRecord)
 
-            if socket == nil {
-                if !disconnected {
-                    reconnect()
-                }
-            } else {
-                sendOperationAsync(.Subscribe(requestId: subscriptionRecord.requestId, query: query))
+        if socket == nil || socket?.readyState != .OPEN{
```

Comment:
Also, we might as well not do anything if the socket is in `CONNECTING` state, since we are going to destroy and re-create the socket, but we don't have to since the subscriptions were not sent yet.